src/threadpool-pthreads.c

```diff
@@ -439,8 +449,18 @@ struct pthreadpool* pthreadpool_create(size_t threads_count) {
 #endif
 
 	if (threads_count == 0) {
-		threads_count = (size_t) sysconf(_SC_NPROCESSORS_ONLN);
+#		if defined(_SC_NPROCESSORS_ONLN)
```

Comment:
Please indent the whole `#if` statement, not just the `if` part of it (similarly to `#if`/`#else` blocks elsewhere in this file).

Reply:
@mattn

Reply:
sorry. I was drinking coffee with I intent that I was already fixed it.